viewer/packages/pointclouds/src/potree-three-loader/rendering/shaders/pointcloud.vert

```diff
@@ -600,16 +600,24 @@ void main() {
 	#endif
 
         if (isClipped((modelViewMatrix * vec4(position, 1.0)).xyz)) {
-                 gl_Position = vec4(1000.0, 1000.0, 1000.0, 1.0);
+                gl_Position = vec4(1000.0, 1000.0, 1000.0, 1.0);
         }
 
-        if (objectId > 0.0) {
-            int lutX = int(objectId) % OBJECT_STYLING_TEXTURE_WIDTH;
-            int lutY = int(objectId) / OBJECT_STYLING_TEXTURE_WIDTH;
-            vec3 colorTexel = texelFetch(objectIdLUT, ivec2(lutX, lutY), 0).rgb;
+        int lutX = int(objectId) % OBJECT_STYLING_TEXTURE_WIDTH;
```

Comment:
Could there be some shared logic with styled CAD objects here? Should be identical, except that the objectIds differ.

Reply:
Yes, it is similar, but I don't see how we can share logic between the two packages without having to resort to extremely long and brittle `glslify` import strings. Maybe we could refactor to gather shaders in the same spot, but sounds like a job for another PR.

Reply:
Tracking it as an issue here: https://cognitedata.atlassian.net/browse/REV-433